client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -34,7 +34,9 @@ export function* createSite(
 	username: string,
 	languageSlug: string,
 	bearerToken?: string,
-	isPublicSite = false
+	visibility: number = isEnabled( 'gutenboarding/public-coming-soon' )
+		? Site.Visibility.PublicNotIndexed
```

Comment:
Is this defaulting sites to the not indexed option?

Reply:
Yep. This default argument is never actually used (a value is always passed it), but sites in coming soon mode start as unindexed.

Reply:
The setting will actually come from here: https://github.com/Automattic/wp-calypso/pull/45428/files#diff-6be59baa71efc91a49a8e47a21563362R55